development/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM php:7.3-fpm-buster
```

Comment:
Is there a point for us to specifically use the `buster` version? In other words: Would going with `php:7.3-fpm` be specific enough for us.

I spotted from the production setup that DDF already provides a PHP 7.3 FPM image.

Reply:
> 👍 for improving the development setup.
> 
> I have some comments about the current implementation for you to take a look at.
> 
> Also I tried checking out the branch and running `docker-compose up`. Here the database just kept restarting with the following in the logs. Am I missing something?
> 
> ```
> laravel-db | 2022-01-18 12:33:09+00:00 [Note] [Entrypoint]: Switching to dedicated user 'mysql'
> laravel-db | 2022-01-18 12:33:09+00:00 [Note] [Entrypoint]: Entrypoint script for MySQL Server 5.7.36-1debian10 started.
> laravel-db | 2022-01-18 12:33:10+00:00 [ERROR] [Entrypoint]: Database is uninitialized and password option is not specified
> laravel-db |     You need to specify one of the following:
> laravel-db |     - MYSQL_ROOT_PASSWORD
> laravel-db |     - MYSQL_ALLOW_EMPTY_PASSWORD
> laravel-db |     - MYSQL_RANDOM_ROOT_PASSWORD
> laravel-db | 2022-01-18 12:33:11+00:00 [Note] [Entrypoint]: Switching to dedicated user 'mysql'
> laravel-db | 2022-01-18 12:33:11+00:00 [Note] [Entrypoint]: Entrypoint script for MySQL Server 5.7.36-1debian10 started.
> laravel-db | 2022-01-18 12:33:11+00:00 [ERROR] [Entrypoint]: Database is uninitialized and password option is not specified
> laravel-db |     You need to specify one of the following:
> laravel-db |     - MYSQL_ROOT_PASSWORD
> laravel-db |     - MYSQL_ALLOW_EMPTY_PASSWORD
> laravel-db |     - MYSQL_RANDOM_ROOT_PASSWORD
> laravel-db exited with code 1
> ```
> 
> In general it would be good to have [setup/usage documented in the README](https://github.com/danskernesdigitalebibliotek/ddb-material-list/blob/develop/README.md#installation).

I think it is probably because you did not copy the .env.example to .env as stated in the README file.
But yes it is a good idea indeed to write in the README how you are supposed to use the docker-compose setup. I'll do that.